linkage-monitor/src/main/java/com/google/cloud/tools/dependencies/linkagemonitor/LinkageMonitor.java

```diff
@@ -111,6 +121,29 @@ public static void main(String[] arguments)
     }
   }
 
+  private static CommandLine parseCommandLine(String[] arguments) {
+    Options options = new Options();
+    options.addOption("send-analytics", false, "track results from each run");
+    CommandLineParser parser = new DefaultParser();
+    try {
+      CommandLine commandLine = parser.parse(options, arguments);
+      if (commandLine.getArgList().size() != 1) {
+        printUsageAndDie();
+      }
+      return commandLine;
+    } catch (ParseException ex) {
+      printUsageAndDie();
+      return null;
+    }
+  }
+
+  private static void printUsageAndDie() {
+    logger.severe(
+        "Usage: java com.google.cloud.tools.dependencies.linkagemonitor.LinkageMonitor"
+        + " --[no]send-analytics com.google.cloud:libraries-bom");
```

Comment:
I don't see option for `--nosend-analytics` (yet?)

Reply:
I'm too used to gnu. Fixed.